src/parser/content-evaluator-module.ts

```diff
@@ -90,23 +130,21 @@ export class ContentEvaluatorModule implements Module {
       });
 
       const rawResponse = String(response.choices[0].message.content);
-      const parsedResponse = JSON.parse(rawResponse) as number[];
-      return parsedResponse.map((o) => new Decimal(o));
+      return JSON.parse(rawResponse) as { [k: string]: number };
     } catch (error) {
       console.error(`Failed to evaluate comment`, error);
-      return [];
+      return {};
```

Comment:
seems weird to return `{}`, if AI evaluation failed we should probably stop execution

Reply:
Ok, any issues with evalutation or openai API propagates the exception, resulting in stopping execution and showing error on github comment.